src/main/java/org/opensearch/ad/feature/SearchFeatureDao.java

```diff
@@ -578,7 +578,7 @@ private Map<Long, Optional<double[]>> parseBucketAggregationResponse(SearchRespo
         List<Aggregation> aggregations = response.getAggregations().asList();
         logger.debug("Feature aggregation result size {}", aggregations.size());
         for (Aggregation agg : aggregations) {
-            List<InternalComposite.InternalBucket> buckets = ((InternalComposite) agg).getBuckets();
+            List<ParsedComposite.ParsedBucket> buckets = ((ParsedComposite) agg).getBuckets();
```

Comment:
@owaiskazi19 @joshpalis shouldn't we document these changes?

Reply:
sure I can add this to the migration documentation